db/migrate/20170923225655_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
```

Comment:
Can we have one of these be not null?

Reply:
Hm, good point, I think we should have `firstName` & `lastName`. Where `lastName` can be optional.

So `firstName` & `email` can't be null